### PR TITLE
Fix publishing of static binaries

### PR DESCRIPTION
**Pull request by nh13**
Fixes: #21 

See the successful publish github action: https://github.com/fulcrumgenomics/tmp-fqgrep-githubactions/actions/runs/4239758449.
See the binaries automatically generated by the publish github action: https://github.com/fulcrumgenomics/tmp-fqgrep-githubactions/releases/tag/v16

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -7,29 +7,129 @@ on:
...
+            arm-linux-gnueabihf-strip \
+            /target/arm-unknown-linux-gnueabihf/release/fqgrep
+
+      - name: Rename binary
+        shell: bash
+        run: |
+          if [ "${{ matrix.os }}" = "windows-2022" ]; then
+            cp "target/${{ matrix.target }}/release/fqgrep.exe" "fqgrep-${{ matrix.target }}"
+          else
+            # The man page is only generated on Unix systems. ¯\_(ツ)_/¯
```
Comment (nh13) [inline .github/workflows/publish.yml:125]:
```suggestion
```

**Comment by jma1991** [comment]
[fqgrep-x86_64-apple-darwin](https://github.com/fulcrumgenomics/tmp-fqgrep-githubactions/releases/download/v16/fqgrep-x86_64-apple-darwin) runs with no errors on MacBook Air (M1, Ventura 13.0.1)

**Comment by bnovak32** [comment]
_fqgrep-x86_64-apple-darwin_ runs without errors on a MacBook Air (M2, Ventura 13.2.1)

Using docker on the same computer, I can also run _fqgrep-x86_64-unknown-linux-musl_ with no issues (docker images based on alpine and ubuntu both work).

When I try to run _fqgrep-arm-unknown-linux-gnueabihf_ in an ubuntu container I get this error: "qemu-arm: Could not open '/lib/ld-linux-armhf.so.3': No such file or directory"

**Comment by nh13** [comment]
@bnovak32 I think the arm release you need to use the `arm32v7/ubuntu` docker image (32-bit).  I don't have an arm64 target (e.g. `aarch64-unknown-linux-gnu`).  I'd be glad for a PR if folks want one.

**Comment by bnovak32** [comment]
> @bnovak32 I think the arm release you need to use the `arm32v7/ubuntu` docker image (32-bit). I don't have an arm64 target (e.g. `aarch64-unknown-linux-gnu`). I'd be glad for a PR if folks want one.

I think it would be great to have an arm64 version because that's what you get when running docker on an m1/m2 Mac. I try to always build amd64/arm64 multi-platform docker images to easily transfer workflows across different systems. I can just install from source, though, so it's not a big deal.

**Comment by nh13** [comment]
@bnovak32 if you want to test out a `aarch64-unknown-linux-gnu`, it's a simple as creating a new repo (not forking), pushing the main branch here up there, then update the publish with the github actions (add the `aarch64-unknown-linux-gnu` target), then pushing a tag to kick off the publish github action.

**Pull request merged**